oeps/best-practices/oep-0049-django-app-patterns.rst

```diff
@@ -267,7 +267,35 @@ Do note that even if you expose your tasks through ``api.py`` to be used by othe
 still be configured with the ``tasks`` import name, as the celery identifier for your task (as set by the celery
 decorator) is based off the original file.
 
+docs/decisions/0001-purpose-of-this-repo.rst / 0001-purpose-of-this-app.rst
+===========================================================================
+
+This should be an architectural decision record (ADR) describing the decision behind adding the app. Future ADRs should also be placed in this directory. See more about what should go in ADRs in the `ADRs section of OEP-0019`_
+
+If this is a repository that only contains a single app, the information should be in a file called `0001-purpose-of-this-repo.rst`. There should be a `0001-purpose-of-this-app.rst` ADR in application subdirectory (in its own docs/decisions subdirectory) that just points to the repository-level one.
```

Comment:
1. I stick with my original proposal to update the title of this section to `docs/decisions/0001-purpose-of-this-app.rst`. All of the other sections are relative to the app directory.
2. I'm wondering if this could be simpler, and not necessarily require (0001-purpose-of-this-repo.rst), since this OEP isn't about that. How do you feel about something like the following?
```
If this is the only app in the entire repo, this ADR could refer to a ``0001-purpose-of-this-repo.rst`` in the repository-level docs/decisions directory.
```

Reply:
I might replace "could" with "should" but that's fine by me

Reply:
@rgraber: optional: You could reference https://github.com/openedx/edx-cookiecutters#3-todos-after-running-cookiecutter if you'd like.

Reply:
I approve once this has been updated.

Reply:
I recommend "[could/should] be a stub that links to" over "should refer to". I think the latter leaves room for misintepretation (ie "do I have to write a whole new ADR, that happens to contain a link to the repo ADR?")